<launch>

    <arg name="zed_odom_topic" default=""/>
    <arg name="out_odom_topic" default=""/>

    <arg name="static_frame" default="" />
    <arg name="moving_frame" default="" />

    <arg name="output" default="log" />

    <node name="convert_zed_odom" pkg="convert_zed_odom" type="convert_zed_odom_node" output="$(arg output)" >
      <remap from="~/odom_in" to="$(arg zed_odom_topic)"/>
      <remap from="~/odom_out" to="$(arg out_odom_topic)"/>
      <param name="static_frame" type="str" value="$(arg static_frame)" />
      <param name="moving_frame" type="str" value="$(arg moving_frame)" />
    </node>

</launch>
